sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogManager.scala

```diff
@@ -27,6 +27,7 @@ import org.apache.spark.sql.internal.SQLConf
  * A thread-safe manager for [[CatalogPlugin]]s. It tracks all the registered catalogs, and allow
  * the caller to look up a catalog by name.
  */
+private[sql]
```

Comment:
do we not want to make this API public, similar to the catalog implementation we had before?

Reply:
yea, `CatalogManager` is a completely internal stuff.